modules/debug_cmd/debug_cmd.c

```diff
@@ -172,7 +177,9 @@ struct fileinfo_st {
 	struct ausrc_prm prm;
 	size_t sampc;
 	struct tmr tmr;
-	bool   finished;
+	RE_ATOMIC bool finished;
+	struct re_printf *pf;
```

Comment:
taking a copy of this pointer may lead to undefined behaviour.

In this module, we dont know how `pf` is declared in the application.

Reply:
Also, may become obsolete.